content/docs/components/serving/overview.md

```diff
@@ -182,7 +182,7 @@ Further information:
  * KFServing:
    * [Kubeflow documentation](/docs/components/serving/kfserving/)
    * [GitHub repository](https://github.com/kubeflow/kfserving)
-   * [Community]/docs/about/community/)
+   * [Community](https://www.kubeflow.org/docs/about/community/)
```

Comment:
I think that should work only add "(",  thanks, Jane

Reply:
Thanks @jinchihe! 
I have modified the file. Please review/approve. Let me know if further change is needed.